terraspace/tfsec.yml

```diff
@@ -0,0 +1,11 @@
+# Encryption at rest rules are currently excluded (https://filecoinproject.slack.com/archives/C02BZPRS9HP/p1662043881772469)
+exclude:
+- aws-ec2-enable-at-rest-encryption
+- aws-sns-enable-topic-encryption
+- aws-sqs-enable-queue-encryption
+- aws-s3-enable-bucket-encryption
+- aws-s3-encryption-customer-key
+- aws-dynamodb-enable-at-rest-encryption
+- aws-dynamodb-table-customer-key
+- aws-cloudwatch-log-group-customer-key
+- aws-ecr-repository-customer-key
```

Comment:
I'm on the fence about this.  Should we enable global exclusions, or be specific and specify these exclusions at a per-resource level?  I know the latter is more work.  I'm concerned that if we turn it off globally and introduce something which may be eligible for encryption we will end up missing out on one advantage of having these rules enabled at a global level.

Reply:
Based on what we have agreed about encryption at rest at this project phase, it will be a bad experience if we have to add new ignores every time we introduce a new queue, table, etc.